compiler/src/dotty/tools/dotc/typer/Namer.scala

```diff
@@ -1029,10 +1029,13 @@ class Namer { typer: Typer =>
 
         def addWildcardForwarders(seen: List[TermName], span: Span): Unit =
           for mbr <- path.tpe.membersBasedOnFlags(required = EmptyFlags, excluded = PrivateOrSynthetic) do
-            val alias = mbr.name.toTermName
-            if !seen.contains(alias)
-               && mbr.matchesImportBound(if mbr.symbol.is(Given) then givenBound else wildcardBound)
-            then addForwarder(alias, mbr, span)
+            if !mbr.symbol.isSuperAccessor then
+              // Scala 2 superaccessors have neither Synthetic nor Artfact set, so we
```

Comment:
Alternatively I guess we could set these flags for superaccessors in Scala2Unpickler, that way we're sure they're treated like the dotty ones.

Reply:
I'd consider that if we find other areas where the missing flag for super accessors messes things up. As it is now, we exercise the test a lot less often than if we have to test every symbol read from Scala 2 for whether it is a super accessor.